lib/paginate-request.js

```diff
@@ -1,7 +1,7 @@
 const { isNil, merge  } = require('omnibelt');
-module.exports =  async (request, query) => {
+module.exports =  async (request, query, shouldUseOffset = false) => {
   let items = [];
-  if (!query.perPage) { query.perPage = 1000; }
+  if (!query.perPage) { query.perPage = query.limit || 1000; }
```

Comment:
so you seem to e creating a query here that has both `limit` and `perPage`, and not actually defaulting `limit` when it is not set and `shouldUseOffset` is true

Reply:
Talked with Erin about this -
we're never actually passing in perPage or page to this function, so there's not really a reason to check if they have a value here (because they never will). 
Going to just set them to 1000 and 0 respectively, and construct an obj with them based on the value of shouldUseOffset